images/scripts/rhel.setup

```diff
@@ -34,8 +34,39 @@ set -x
 
 if [ ! -f "$SKIP_REPO_FLAG" ]; then
     # Configure repositories.
+    if [[ "$SERVER_REPO_URL" && "$EXTRAS_REPO_URL" ]]; then
```

Comment:
OK, fixed. Should I also use the POSIX syntax in `image-create-rhel-compose` to keep it consistent with other scripts?

Reply:
Please use `[ -n "$var" ]`; just `[ "$var" ]` is unsafe in general. Even though the values should always start with a letter, let's be correct just in case. If you want to POSIXify image-create-rhel-compose, that's fine for me, but it's less important.

Reply:
Ah, of course. Fixed :).